scanrepository/scanrepository.go

```diff
@@ -276,11 +276,18 @@ func (cfp *ScanRepositoryCmd) fixIssuesSinglePR(vulnerabilitiesMap map[string]ma
 // else, return the error
 func (cfp *ScanRepositoryCmd) handleUpdatePackageErrors(err error) error {
 	var errUnsupportedFix *utils.ErrUnsupportedFix
-	if errors.As(err, &errUnsupportedFix) {
+	var errNoChangesToCommit *utils.ErrNoChangesToCommit
+
+	switch {
+	case errors.As(err, &errUnsupportedFix):
 		log.Debug(strings.TrimSpace(err.Error()))
 		return nil
```

Comment:
We can return nil at the end of the function to reduce code duplication (for both cases, leave the "return err" only for default cases)